docs/BingMaps_API_Key_Guidance.md

```diff
@@ -11,5 +11,4 @@ An account with Bing Maps can be created using the [Bing Maps Portal](https://ww
 
 ### **Using your API Key**
 
-#### To do: Create Local and Docker build instructions.
-Once a key is obtained, please insert your key and any arbitrary username in the [ISDcreator-webapp-keys](/private-resources/js/ISDcreator-webapp-keys.js) file and build your updated ConnectedVCS Tools using either the [Local](/docs/) or [Docker](/docs/) build instructions.
+Once a key is obtained, please insert your key and any arbitrary username in the [ISDcreator-webapp-keys](/private-resources/js/ISDcreator-webapp-keys.js) file and build your updated ConnectedVCS Tools using either the [Local](/docs/Local_Instructions.md) or [Docker](/docs/Docker_Instructions.md) build instructions.
```

Comment:
It seems like Bing Map API keys are deprecated and new users will not be able to create these keys.

![image](https://github.com/user-attachments/assets/0c94ae0e-feae-4e41-a55a-4effd1350383)

We may have to add a note here that new keys will be deprecated.

Reply:
Note added. New story will be created to migrate from Bing Maps.